applications/CompressiblePotentialFlowApplication/custom_python/add_custom_strategies_to_python.cpp

```diff
@@ -0,0 +1,57 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Marco Antonio Zuñiga Perez
```

Comment:
Very minor, amend the indentation of the header.

Reply:
The indentation is ok when I try to edit it. I think It's just because of the browser.

Reply:
Are you sure you're not mixing tabs and spaces?

Reply:
I think now it's ok.